napari/utils/shortcuts.py

```diff
@@ -17,6 +17,7 @@
     'napari:focus_axes_down': [KeyMod.Alt | KeyCode.DownArrow],
     'napari:roll_axes': [KeyMod.CtrlCmd | KeyCode.KeyE],
     'napari:transpose_axes': [KeyMod.CtrlCmd | KeyCode.KeyT],
```

Comment:
On main, this shortcut was displayed at the end of the tooltip message, but with this PR we no longer have the `Cmd+T` shortcut displayed. Do we know why that is... pinging @dalthviz because he's been looking at this stuff recently. See below roll axes vs. the transpose button with this PR.

Roll axes has shortcut displayed:

![image](https://github.com/user-attachments/assets/03b19b24-9564-4626-be4c-477c2f78efff)

Transpose, with this PR, doesn't:

![image](https://github.com/user-attachments/assets/bc8a89c4-b1f7-44a1-a67f-012c0a27ff42)

Reply:
Gave a quick check and I think that's caused since over https://github.com/napari/napari/pull/7052/files/a8c1445e52bbec4e4bb4c33ca5d066e5f2df2e28#diff-91ff0176aad5008b16e25075ab1de158e640bc75a256a179f58a46670fc4970eR145 the `action` kwarg was removed so the logic at

 https://github.com/napari/napari/blob/bed967f81fcef2b86bd83d5c1b741d98cc2d91fe/napari/utils/action_manager.py#L232-L235

never gets called

Reply:
Right, the button isn't linked to the action. 

I couldn't think of a way to have two actions linked to the button, but one with alt-click.

Reply:
That looks excellent!

Reply:
Here a branch with the proposed changes: https://github.com/dalthviz/napari/commit/75dccf6386b11907803ec2a09750dd0682876b7f I also added there a way in the button class definition to pass the `extra_tooltip_text` kwarg to the action manager `bind_button` method call. The tooltip end up looking something like:

![imagen](https://github.com/user-attachments/assets/a043f0c8-af0b-4848-b039-13417414e12e)

Could it be worthy for me to try to open a PR over your fork with those changes?

Reply:
That tooltip looks good!
I wonder if we can show the keyboard shortcut in the alt-click action tooltip too? 🤔 

Anyhow, whatever is the path of least resistance for you. If you want to make a PR to this PR branch, that works. Or I can (try to) cherry pick or whatever.
Thanks! You're a 🧙

Reply:
Makes sense. Many buttons could have 2 functions, one with alt--moving into the toolbar-style buttons I guess.